bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -18,6 +18,13 @@ pub struct ExecutionResult {
     create_address: ffi::evmc_address,
 }
 
+pub struct ExecutionContext<'a> {
+    rev: &'a ffi::evmc_revision,
+    msg: &'a ffi::evmc_message,
+    ctx: &'a mut ffi::evmc_context,
+    inst: &'a mut ffi::evmc_instance,
```

Comment:
I'd prefer to use long names (or at least not 3-4 character abbreviations).

Reply:
i plan to scrap these members anyway in favor of a higher level representation of all the context data. will keep it in mind.